libs/vkd3d/vkd3d_private.h

```diff
@@ -2796,6 +2801,10 @@ struct d3d12_command_list_sequence
     unsigned int active_non_inline_running_queries;
     bool uses_dgc_compute_in_async_compute;
 
+    /* Number of draws, dispatches, copies etc. Used to fuse barrier-only
+     * command buffers for staggered submissions. */
+    uint32_t estimated_cost;
```

Comment:
I think we'll want a cost estimate for the init command list as well. Preprocess generation work can be considered LOW cost for example, but mostly it's rather trivial stuff I think that shouldn't be considered to have cost at all I think?